src/plugins/integrations/AWSIntegration.ts

```diff
@@ -267,7 +272,6 @@ class AWSIntegration implements Integration {
                   KinesisRequestTypes[operationName] : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done